app/3.0/docs/glossary.md

```diff
@@ -23,50 +20,28 @@ The [Polymer CLI](#polymer-cli) is designed to work with apps that use an app sh
 See [Instant Loading Web Apps with an Application Shell 
 Architecture](https://developers.google.com/web/updates/2015/11/app-shell) for more information.
 
-## B
-### Bower
-Bower is a package manager for web projects. Bower automates the installation of packages like 
-custom elements and JavaScript libraries. Polymer distributes its core library and elements via 
-Bower. See [bower.io](https://bower.io) for more information.
+### bundling
 
-### bower.json
-`bower.json` is a manifest used by Bower for configuring packages for use as dependencies. See [the 
-`bower.json` specification](https://github.com/bower/spec/blob/master/json.md) for more information.
+Bundling concatenates an app's scripts and resources to reduce the number of network requests it takes to load dependencies. 
 
-## C
-### crisper
-crisper extracts inline scripts from an HTML file. crisper is useful for separating JavaScript from 
-HTML in order to deploy in an environment that uses CSP (Content Security Policy).
+By contrast, if both server and browser support HTTP/2 (a newer protocol that permits concurrent resource requests), it can be optimal to leave resources unbundled. Unbundled builds permit lazy loading of dynamic dependencies. By default, Polymer apps are unbundled.
 
 ## D
 ### dependency
 
 *dependency* may refer to:
 
-* A package dependency, such as a Bower or npm package that your project uses.
+* A package dependency, such as an npm package that your project uses.
 * A specific resource that is required by another, such as HTML files or JavaScript libraries. See 
 also: [dynamic dependency](#dynamic-dependency), [static dependency](#static-dependency), 
 [transitive dependency](#transitive-dependency).
 * A data dependency in the context of the Polymer data system: a piece of data that an observer, 
 computed property or computed binding depends on.
 
-### dependency variants
-
-Dependency variants are named sets of Bower [package dependencies](#dependency) at specific 
-versions which are different (usually more specific) than the default dependencies. Variants are 
-specified in [`bower.json`](#bower-json).
+### dynamic import()
+A [dynamic JavaScript `import()` statement](#https://developers.google.com/web/updates/2017/11/dynamic-import) loads the bindings exported by another module. In contrast to static [import](#import) statements, dynamic imports load their targets at runtime. 
```

Comment:
"Loads the binding" is a little specific. Loads the module is fine. Both static and dynamic import load at runtime, but dynamic imports load based on code execution, and can take a computed string.